handler/cantabular_metadata_export.go

```diff
@@ -284,24 +292,33 @@ func (h *CantabularMetadataExport) exportCSVW(ctx context.Context, e *event.CSVC
 	return &d, nil
 }
 
-func (h *CantabularMetadataExport) generateTextFilename(e *event.CSVCreated) string {
+func (h *CantabularMetadataExport) generateTextFilename(e *event.CSVCreated, isCustom bool) string {
 	var prefix, suffix string
 
+	datasetID := e.DatasetID
+	if isCustom {
+		datasetID = "custom"
+	}
+
 	if len(e.FilterOutputID) == 0 {
 		prefix, suffix = "datasets/", ".txt"
 	} else {
 		prefix = fmt.Sprintf("datasets/%s/", e.FilterOutputID)
 		suffix = fmt.Sprintf("-%s.txt", h.generate.Timestamp().Format(time.RFC3339))
 	}
 
-	fn := fmt.Sprintf("%s-%s-%s", e.DatasetID, e.Edition, e.Version)
+	fn := fmt.Sprintf("%s-%s-%s", datasetID, e.Edition, e.Version)
 
 	return prefix + fn + suffix
 }
 
-func (h *CantabularMetadataExport) generateCSVWFilename(e *event.CSVCreated) string {
+func (h *CantabularMetadataExport) generateCSVWFilename(e *event.CSVCreated, isCustom bool) string {
 	var prefix, suffix string
 
+	datasetID := e.DatasetID
+	if isCustom {
```

Comment:
I'm pretty sure the ticket reads 'replace the datastID, edition and version with custom' rather than just the datasetID